teachertool/src/components/CriteriaResultEntry.tsx

```diff
@@ -44,6 +45,31 @@ const CriteriaResultNotes: React.FC<CriteriaResultNotesProps> = ({ criteriaId })
     );
 };
 
+interface CriteriaResultErrorProps {
+    criteriaInstanceId: string;
+    error: string;
+}
+
+const CriteriaResultError: React.FC<CriteriaResultErrorProps> = ({ criteriaInstanceId, error }) => {
+    return (
+        <div className={css["result-error"]}>
```

Comment:
I believe we want it excluded from the printed copy.
```suggestion
        <div className={classList(css["result-error"], "no-print")}>
```

Reply:
![image](https://github.com/user-attachments/assets/f6311551-f3ca-40bf-a60b-ffe3866118d6)